### PR TITLE
Implement Preview panel

### PR DESCRIPTION
**Pull request by ostap1010**
### Description of the Change
Implement preview side panel instead of app preview in the tab and 4-split view.

### Github Issues

#311

### CODE REVIEW

**Comment by javier-tarazaga** [comment]
@ostap1010 couple of quick comments while testing:
1. The panels now need the possibility to change the width as otherwise is really hard with a fixed width with the preview panel (the other did not matter that much as the info in the panel was deterministic)
2. I tried the interact screen against Rinkeby (using the voting template), and the connection with Metamask seems broken.

File: src/components/projecteditor/panes.js [hunk 1]
```diff
@@ -308,16 +308,10 @@ class Panes extends Component {
     render() {
         const panes = this.renderPanes();
 
-        const { isActionPanelShowing } = this.props;
-
         return (
             <div
-                key="panes"
```
Comment (Bashlund) [inline src/components/projecteditor/panes.js]: Is it 100% safe to remove this?

Comment (ostap1010) [reply src/components/projecteditor/panes.js]: Seem to work ok. It is very strange there was a "key" attribute, as keys are designed only for rendering lists/arrays. And it is the case here.

**Comment by ostap1010** [comment]
1. We were talking with Javi about it and the we just decided to keep it to look like as a browser and for future development. @javier-tarazaga was there any other reason? Should I better remove that one?
2. It does not automatically refresh the current app view(when in it is in the separate window). And it is quite a big feature to support that. Maybe we can have it as a separate PR?
3. @javier-tarazaga told me to remove that one :)

File: src/components/projecteditor/panes.js (hunk 1 above)
Comment (Bashlund) [reply src/components/projecteditor/panes.js]: Ok, good then :)

**Comment by javier-tarazaga** [comment]
1. The idea behind this is that we emulate the feel of being a browser as later we will need to support back/fwd actions, as a more complex Dapps might actually have the needs for routing (by changing URLs). Was disabled right now on purpose to not have means to copy/paste it.  

2. That's Ostap's ballpark :) 

3. We could add it back if you think is really absolutely necessary. We could add it with something like this.  @Bashlund what do you think? 
screenshot 2019-01-07 at 09 57 01

**Comment by Bashlund** [comment]
1. If we expose the URL I'm thinking people actually think they can share it, and they will get disappointed or confused. Your call.
2. Yeah, let's take it in another PR :)
3. We can skip for now, but I would think it's important for people to be able to simulate when MM is not available.

**Pull request merged**